infrastructure/identity-aware-proxy/createSelfSignedCert.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
I had imagined most customers running this on their cloud shell but this is a great point since it may not always be in the cloud shell. Good catch, thanks!